File: app/src/main/java/com/dani_chuks/andeladeveloper/starwars/data/models/PlanetList.java

package com.dani_chuks.andeladeveloper.starwars.data.models;

import com.dani_chuks.andeladeveloper.starwars.data.models.entities.Planet;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class PlanetList {

    @SerializedName("count")
    @Expose
    private Integer count;
    @SerializedName("next")
    @Expose
    private String next;
    @SerializedName("previous")
    @Expose
    private Object previous;
    @SerializedName("results")
    @Expose
    private List<Planet> planet = null;

    public Integer getCount() {
        return count;
    }

    public void setCount(Integer count) {
        this.count = count;
    }

    public String getNext() {
        return next;
    }

    public void setNext(String next) {
        this.next = next;
    }

    public Object getPrevious() {
        return previous;
    }

    public void setPrevious(Object previous) {
        this.previous = previous;
    }

    public List<Planet> getPlanet() {
        return planet;
    }

    public void setPlanet(List<Planet> planet) {
        this.planet = planet;
    }

}

File: app/src/main/java/com/dani_chuks/andeladeveloper/starwars/data/models/PlanetList.kt
package com.dani_chuks.andeladeveloper.starwars.data.models

import com.dani_chuks.andeladeveloper.starwars.data.models.entities.Planet
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PlanetList {

    @SerializedName("count")
    @Expose
    var count: Int? = null
    @SerializedName("next")
    @Expose
    var next: String? = null
    @SerializedName("previous")
    @Expose
    var previous: Any? = null
    @SerializedName("results")
    @Expose
    var planet: List<Planet>? = null

}
